ahnlich/server/src/main.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
+#![allow(clippy::size_of_ref)]
```

Comment:
@deven96 allowing the size of ref for now. We would need to come up with the right implementation for calculating the size of the Store

Reply:
Yeah... all the sizing already feels very off